DATA/UNIVERSE/SYSTEMS/Br01/Br01.ini

```diff
@@ -10289,3 +10289,215 @@ parent = Br01_08
 visit = 128
 Archetype = jump_hole_invisible
 goto = Br01, br01_from_start_unlawful, gate_tunnel_bretonia
+
+;;Mineable Gas Pockets
+
+[Object]
+nickname = mineable_small_co2_Br01_01

```

Comment:
This name got me. I thought first it's a mining base, small sized, of the Corsairs. But I understand it.

Reply:
corsaris is just c nor co

co is even in vanilla used for companies not corsairs